assembl/graphql/schema.py

```diff
@@ -323,6 +323,7 @@ class Video(graphene.ObjectType):
 class IdeaInterface(graphene.Interface):
     num_posts = graphene.Int()
     num_contributors = graphene.Int()
+    num_children = graphene.Int()
```

Comment:
Could you please add a simple test, by adding numChildren in the following test tests/views_tests/test_graphql_ideas.py:test_graphql_get_all_ideas for example. Thanks!